packages/icons/icons.json

```
@@ -891,21 +891,21 @@
     {
         "displayName": "Table",
         "iconName": "th",
-        "tags": "index, rows, columns, agenda, list, spreadsheet",
+        "tags": "index, rows, columns, agenda, list, spreadsheet, table, sheet, entry",
         "group": "table",
         "codepoint": 62186
     },
     {
         "displayName": "Table: list",
         "iconName": "th-list",
-        "tags": "index, rows, list, order, series",
+        "tags": "index, rows, list, order, series, calendar, table",
```

```suggestion
        "tags": "index, rows, list, order, series, table",
```

we have a strictly better icon for "calendar", let's not conflate them